specification/dnsresolver/Network.Management/client.tsp

```diff
@@ -0,0 +1,7 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+
+@@clientName(Microsoft.Network.ActionType, "DnsSecurityRuleActionType");
```

Comment:
@haolingdong-msft  Are you sure this change in scope applies to all languages? I noticed that for JavaScript, this change was marked as a breaking change.
<img width="423" height="25" alt="image" src="https://github.com/user-attachments/assets/e253efd1-adba-4ea0-a0b8-2d80311efed3" />
<img width="428" height="28" alt="image" src="https://github.com/user-attachments/assets/1a428269-192a-469a-a7b4-212b71d6ee1d" />

Reply:
It is an issue happens before. refer to the description in this pr: https://github.com/Azure/azure-rest-api-specs/pull/35960